github/ci/prow-deploy/files/jobs/kubevirt/test-benchmarks/test-benchmarks-presubmits.yaml

```
@@ -0,0 +1,27 @@
+presubmits:
+  kubevirt/libguestfs-appliance:
+  - name: pull-test-benchmarks
+    branches:
+      - main
+    always_run: true
+    optional: false
```

For this first one it should be ok, but please bear in mind that in general it is advised to introduce new jobs with the config changed like this
```suggestion
    always_run: false
    optional: true
    skip_report: true
```
This implies that at first jobs are triggered manually until they are stable and also do not report in order to not confuse users with a possibly bad ci signal.